### DISCUSSION TITLE
Have a menu completion like in zsh

### DISCUSSION THREAD

**Discussion by TimotheeMathieu** [xonsh/xonsh#4880] [Q&A] [answered]
Hello,
I just migrated from zsh and this is a very nice shell but I want to keep the zsh completion menu. Is this possible ? 

I tried to set `$COMPLETIONS_DISPLAY = 'readline'` but this did not seem to change anything for me.

My question is linked to this issue I guess : https://github.com/xonsh/xonsh/issues/2364 . The author say it is solve but he do not say how he did it. I did not manage to have something similar to what is in the video with xonsh, I always get a pop-up menu while I prefer a menu below the current line as in the youtube video in the issue.

Thanks.

**Comment by gforsyth** [comment, marked as answer]
Hey @TimotheeMathieu -- 
I _think_ that what you want is to set `$COMPLETIONS_DISPLAY="multi"` to match `zsh` default completion style.
The completion menu by default will have a different background color than the terminal background.

(the colors can be changed by changing or editing styles which is a separate topic)

This also assumes that you are using the  `prompt_toolkit` backend -- you can check this by looking at the `shell type` field in the output of `xonfig`

**Reply by TimotheeMathieu** [reply]
Thanks, it works. 
With 
```
$COMPLETIONS_DISPLAY = 'multi'
$XONSH_STYLE_OVERRIDES["completion-menu"] = "bg:#000000 #fff" 
$XONSH_STYLE_OVERRIDES["Token.PTK.CompletionMenu.Completion.Current"] = "bg:#555555 #fff"
```
I get something similar. 
There are still differences in the size of the completion menu but I will go with this.
